firefox-ios/Client/Application/AppDelegate+SyncSentTabs.swift

```diff
@@ -46,4 +47,12 @@ class AppSendTabDelegate: SendTabDelegate {
             }
         }
     }
+
+    func closeTabs(for urls: [URL]) {
+        Task {
+            for url in urls {
```

Comment:
(I left a comment below about passing the list all the way down to the `TabManager`, so that it can remove them all at once).